modules/c++/math/include/math/Constants.h

```diff
@@ -23,6 +23,31 @@
 #ifndef __MATH_CONSTANTS_H__
 #define __MATH_CONSTANTS_H__
 
+#include <sys/sys_config.h>
+
+#ifdef __CODA_CPP11
+#include <math.h>
+namespace math
+{
+struct Constants
```

Comment:
I'm not quite sure what/ if you are asking, but I removed the double definition whenever __CODA_CPP11 is defined.